core/src/reporter/reporter.ts

```diff
@@ -66,3 +69,24 @@ export function reporter(state: State, logger: Logger) {
   logger.debug('Reporter job built')
   return wrapper
 }
+
+export function dataFeedReporter(state: State, logger: Logger) {
+  const reporterAggregateQueue = new Queue(REPORTER_AGGREGATOR_QUEUE_NAME, BULLMQ_CONNECTION)
+  async function wrapper(job: Job) {
+    const splittedJobId = job.id?.split('-')
```

Comment:
I think we should start commenting more the functions about what they accomplish. It is much easier to understand the expected outcome when we show example in comments as well.